utils/api.js

```diff
@@ -27,3 +26,25 @@ export const fetchCommentsByArticle = (article_id) => {
     return res.data;
   });
 };
+
+export function fetchTopics() {
+  return api
+    .get("/topics")
+    .then(({ data }) => {
+      return data.topics;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+}
+
+export function fetchArticlesByTopic(topic) {
+  return api
```

Comment:
The api is set up to do this on your behalf. The front end shouldn't filter if it doesn't need to. Put the load on the server instead of the browser and make a queried request to your API.

Reply:
Unfortunately, my API doesn't seem to be able to do that
https://gabnbar.onrender.com/api/articles?topic=cooking
Lists all the articles, rather than just the ones about cooking.